cobalt/site/docs/development/setup-android.md

```diff
@@ -33,6 +33,11 @@ return and complete the following steps.
 
     If prompted, read and accept the license agreement.
 
+1.  Install JDK 17. google it for how to. Verify java version is 17 and above.
+    *   Search online: "Install JDK 17 [your operating system]"
+    *   Follow the instructions provided for your operating system.
+    *   Verify: Open a terminal/command prompt and run java -version.  The output should start with "17" or higher.
```

Comment:
How does the build failed if it's trying to build with the wrong JDK? If the error message is clear enough I don't think we need to include a verification step here.

If you still think it we to include verification instructions please follow the other steps in the file and wrap the command in backticks to emphasize it.

Reply:
The error should be obvious.